app/_includes/head.html

```diff
@@ -49,7 +49,6 @@
   <link rel="dns-prefetch" href="//cdn.segment.com">
   <link rel="dns-prefetch" href="//sidecar.gitter.im">
   <link rel="dns-prefetch" href="//buttons.github.io">
-  ##preload_assets##
```

Comment:
@theetrain yep, it was showing in the body after running `npm start` (which in turn runs `gulp`)